### PR TITLE
Separate templates added for issues, feature requests and chores

### LINKED ISSUE #611: Adding separate templates for bugs, feature requests and chores
### Actual behaviour
Right now, there is a common issue template for creating any type of issue.

### Expected behaviour
There should be separate issue templates for bugs, adding new features or for chores such as code optimizations.

### Would you like to work on the issue?
Yes

### PR DESCRIPTION
**Pull request by ShivangiSingh17**
Added separate templates for issues, feature requests and chores.
Fixes #611 

## Type of change
Just put an x in the [] which are valid.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.
- [X] `./gradlew assembleDebug assembleRelease`
- [X] `./gradlew checkstyle`

# Checklist:
- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings

### CODE REVIEW

**Comment by ShivangiSingh17** [comment]
@Swati4star can you please the PR and suggest the required changes, if any.

File: .github/issue_template/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+### Expected behaviour
+A clear and concise description of what you expected to happen.
+
+### Actual behaviour
+Tell us what happens instead
+
+### Screenshots
+If applicable, add screenshots to help explain your problem.
+
+### Would you like to work on the issue?
```
Comment (Swati4star) [inline .github/issue_template/bug_report.md]: add `Tell us if you would like to work on this issue` below this

File: .github/issue_template/chore.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+name: Chore
+about: Issues related to docs, workflow, dependency and others
+
+---
+
+### Describe the chore
+A clear and concise description of what you want to do.
+
+### Would you like to work on the issue?
```
Comment (Swati4star) [inline .github/issue_template/chore.md]: same here

File: .github/issue_template/feature_request.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+---
+
+### Describe the feature you'd like to add
+A clear and concise description of what feature you want to have.
+
+### Screenshots
+Add screenshots to provide context or UI mockup.
+
+### Would you like to work on the issue?
```
Comment (Swati4star) [inline .github/issue_template/feature_request.md]: ..

**Comment by ShivangiSingh17** [comment]
@Swati4star can you please review the PR with the suggested changes made.

**Pull request merged**